### PR TITLE
Update Gen 5 Random Battle

### PR DESCRIPTION
**Pull request by ACakeWearingAHat**
movepool edits made by deliberation with randbats staff experienced in gen 5

this is the last time i'm not using vscode i s2g

### CODE REVIEW

File: data/mods/gen5/random-teams.ts [hunk 1]
```diff
@@ -489,7 +511,7 @@ export class RandomGen5Teams extends RandomGen6Teams {
 			item = 'Light Ball';
 		} else if (species.name === 'Shedinja' || species.name === 'Smeargle') {
 			item = 'Focus Sash';
-		} else if (species.name === 'Unown') {
+		} else if (counter.Physical + counter.Special + counter.Status = 1) {
```
Comment (mia-pi-git) [inline data/mods/gen5/random-teams.ts:514]:
```suggestion
		} else if (counter.Physical + counter.Special + counter.Status === 1) {
```

File: data/mods/gen5/formats-data.ts [hunk 2]
```diff
@@ -2044,7 +2044,7 @@ export const BattleFormatsData: {[k: string]: ModdedSpeciesFormatsData} = {
 		doublesTier: "DUU",
 	},
 	rotomfan: {
-		randomBattleMoves: ["airslash", "hiddenpowergrass", "painsplit", "substitute", "thunderbolt", "trick", "voltswitch", "willowisp"],
+		randomBattleMoves: ["airslash", "painsplit", "thunderbolt", "trick", "voltswitch", "willowisp"],
```
Comment (TheImmortal) [inline data/mods/gen5/formats-data.ts:2047]: SubSplit?

File: data/mods/gen5/formats-data.ts [hunk 3]
```diff
@@ -2659,7 +2659,7 @@ export const BattleFormatsData: {[k: string]: ModdedSpeciesFormatsData} = {
 		tier: "LC",
 	},
 	beartic: {
-		randomBattleMoves: ["aquajet", "iciclecrash", "stoneedge", "superpower", "taunt"],
+		randomBattleMoves: ["aquajet", "encore", "iciclecrash", "stoneedge", "superpower"],
```
Comment (TheImmortal) [inline data/mods/gen5/formats-data.ts:2662]: Encore? Why is that better than Taunt?

File: data/mods/gen5/formats-data.ts [hunk 4]
```diff
@@ -2766,7 +2766,7 @@ export const BattleFormatsData: {[k: string]: ModdedSpeciesFormatsData} = {
 		doublesTier: "DUU",
 	},
 	terrakion: {
-		randomBattleMoves: ["closecombat", "quickattack", "stealthrock", "stoneedge", "swordsdance", "xscissor"],
+		randomBattleMoves: ["closecombat", "earthquake", "quickattack", "stealthrock", "stoneedge", "swordsdance", "xscissor"],
```
Comment (TheImmortal) [inline data/mods/gen5/formats-data.ts:2769]: Remove X-Scissor then since it only needed it for Choice.

File: data/mods/gen5/formats-data.ts (hunk 2 above)
Comment (ACakeWearingAHat) [reply data/mods/gen5/formats-data.ts:2047]: no air slash bad

File: data/mods/gen5/formats-data.ts (hunk 3 above)
Comment (ACakeWearingAHat) [reply data/mods/gen5/formats-data.ts:2662]: slow encore is better than slow taunt

Comment (ACakeWearingAHat) [reply data/mods/gen5/formats-data.ts:2662]: and also everyone in set workshop absolutely despised taunt

**Pull request merged**